index.html

```diff
@@ -22,31 +22,34 @@
   <body>
     <div class="page-container">
       <div class="header1">
-      <header class="headercontainer">
-        <a href="#" class="logo">Megha</a>
-        <nav><i class="fa-solid fa-bars hambrgricon"></i></nav>
-      </header>
-    </div>
-<div class="header2">
-      <header class="headercontainer">
-        <a href="#" class="logo">Megha</a>
-        <nav><ul class="navlist">
-          <li> <a href="#">Portfolio</a></li>
-          <li> <a href="#">About</a></li>
-          <li> <a href="#">Contact</a></li>
-        </ul></nav>
-      </header>
-    </div>
+        <header class="headercontainer">
+          <a href="#" class="logo">Megha</a>
+          <nav><i class="fa-solid fa-bars hambrgricon" id="hmbg"></i></nav>
+        </header>
+      </div>
+      <div class="header2">
+        <header class="headercontainer">
+          <a href="#" class="logo">Megha</a>
+          <nav>
+            <ul class="navlist">
+              <li class="linkitems"><a href="#portfolio">Portfolio</a></li>
+              <li class="linkitems"><a href="#about">About</a></li>
+              <li class="linkitems"><a href="#contact">Contact</a></li>
+            </ul>
+            <i class="fa-solid fa-times closeicon" id="closebtn"></i>
+          </nav>
```

Comment:
- [ ] **RECOMENDED:** Once implemented some of the changes required by the previous reviewer, I think you also generated other issues on your mobile menu such as:
   - When the user clicks (or taps) the close (X) button, the mobile menu disappears, but right now it remains there.